content/02.main-text.md

```diff
@@ -1,24 +1,72 @@
+
 `TODO: need archival issue link`
 In addition, discussing manuscripts in the open provides one form of pre- and post-publication peer review `TODO: define this or provide a reference?`, incentivizing the reviews with potential manuscript authorship.
 However, inviting wide authorship brings many technical and social challenges such as how to fairly distribute credit, coordinate the scientific content, and collaboratively manage extensive reference lists.
 
-We present solutions to these challenges based on our recent experience leading a collaborative review "Opportunities And Obstacles For Deep Learning In Biology And Medicine" [@doi:10.1101/142760].
-Our review attracted {{deep_review_authors}} authors from 20 different institutions who were not determined in advance.
-We wrote entirely in the open without restrictions on who was welcome to contribute.
-Although we requested that some authors participate for their specific expertise, most discovered the manuscript organically through conferences or social media and independently decided to contribute.
-To coordinate this effort, we developed a manuscript writing process using the Markdown language, the [GitHub software development platform](https://github.com/greenelab/deep-review/), and our new [Manubot](https://github.com/greenelab/manubot) tool for automating manuscript generation.
+## Contribution workflow
+
+There are many existing collaborative writing platforms ranging from rich text editors, which support Microsoft Word documents or similar formats, to LaTeX-based systems for  technical writing [@doi:10.1038/514127a] such as [Overleaf](https://www.overleaf.com/) and [Authorea](https://www.authorea.com/).
+These platforms ideally offer version control, multiple permission levels, or other functionality to support multi-author document editing.
+Although they work well for editing, they lack sufficient features for managing a collaborative manuscript and attributing precise credit, which are important for open writing.
+
+We adopted standard software development strategies in order to enable any contributor to edit any part of the manuscript but enforce discussion and review of all proposed changes.
+The GitHub platform provided support for organizing and editing the manuscript.
+We used GitHub _issues_ for organization, opening a new issue for each paper under consideration.
+Within the issue, contributors summarized the research, discussed it (sometimes with the original authors), and assessed its relevance to the review.
+Issues also served as an open to-do list and a forum for debating the main message, themes, and topics of the review.
+
+GitHub and the underlying git version control system [@doi:10.1371/journal.pcbi.1004668; @doi:10.1371/journal.pcbi.1004947] also structured the writing process.
```

Comment:
is `structured` the right word here? It feels more like `aided` but maybe I'm more a prisoner to my tools than I think. :)